playground/main.ts

```diff
@@ -4,6 +4,8 @@ import {
   renderRichText,
   useStoryblokBridge,
   apiPlugin,
+  richTextResolver,
+  newSbRichTextOptions
```

Comment:
@alvarosabu is it possible we need to change this to `StoryblokRichTextOptions`?